movai_core_shared/logger.py

```diff
@@ -128,27 +142,56 @@ def _emit(self, record):
             record: The Python log message data record
 
         """
-        log_tags = {"robot": DEVICE_NAME,
-                    "level": record.levelname,
-                    "service": SERVICE_NAME}
+        log_tags = {
+            "robot": DEVICE_NAME,
+            "level": record.levelname,
+            "service": SERVICE_NAME}
 
         if hasattr(record, "tags"):
             log_tags.update(record.tags)
 
+        syslog_tags = {
+            "appname": SERVICE_NAME,
+            "facility":"console",
+            "host": DEVICE_NAME,
+            "hostname": DEVICE_NAME,
+            "severity": record.levelname
+        }
+
         log_fields = {
             "module": record.module,
             "funcName": record.funcName,
             "lineno": record.lineno,
             "message": record.msg,
         }
 
+        syslog_fields = {
+            "module": record.module,
+            "funcName": record.funcName,
+            "lineno": record.lineno,
+            "facility_code": 14,
+            "message": record.msg,
+            "procid": PID,
+            "severity_code": SEVERETY_CODES_MAPPING[record.levelname],
```

Comment:
did you checked it 
because logging verbosity enum is numbers
like DEBUG = 10
and not "DEBUG"

plesae check that syslog.DEBUG not equal to logging.DEBUG
that can make it easier

Reply:
yes I did checked that, I want to map from text to code because that's what syslog demands.